__tests__/app.test.js

```diff
@@ -7,131 +7,162 @@ const testData = require('../db/data/test-data/index');
 beforeEach(() => seed(testData));
 afterAll(() => db.end());
 
-describe('/api/topics', () => {
-	describe('GET', () => {
-		test('Status 200: returns an array of topics', () => {
-			return request(app).get('/api/topics').expect(200).then((response) => {
-				expect(response.body.topics).toEqual(expect.any(Array));
+describe('\nGOOD ENDPOINTS\n', () => {
+	describe('/api/topics', () => {
+		describe('GET', () => {
+			test('Status 200: returns an array of topics', () => {
+				return request(app).get('/api/topics').expect(200).then((response) => {
+					expect(response.body.topics).toEqual(expect.any(Array));
+				});
 			});
-		});
-		test('Status 200: returns an array of objects with the correct properties', () => {
-			return request(app).get('/api/topics').expect(200).then((response) => {
-				const allTopics = response.body.topics;
-				allTopics.forEach((topic) => {
-					expect(topic.hasOwnProperty('slug')).toBe(true);
-					expect(topic.hasOwnProperty('description')).toBe(true);
+			test('Status 200: returns an array of objects with the correct properties', () => {
+				return request(app).get('/api/topics').expect(200).then((response) => {
+					const allTopics = response.body.topics;
+					allTopics.forEach((topic) => {
+						expect(topic.hasOwnProperty('slug')).toBe(true);
+						expect(topic.hasOwnProperty('description')).toBe(true);
+					});
 				});
 			});
 		});
 	});
-});
-
-describe('/api/notARoute', () => {
-	describe('GET ERROR', () => {
-		test('Status 404: Not Found', () => {
-			return request(app).get('/api/notARoute').expect(404);
+	describe('/api/articles', () => {
+		describe('GET', () => {
+			test('Status 200: returns an array of articles', () => {
+				return request(app).get('/api/articles').expect(200).then((response) => {
+					expect(response.body.articles).toEqual(expect.any(Array));
+				});
+			});
+			test('Status 200: returns an array of objects with the correct properties and value types', () => {
+				return request(app).get('/api/articles').expect(200).then((response) => {
+					const allArticles = response.body.articles;
+					allArticles.forEach((article) => {
+						expect(article).toHaveProperty('author');
+						expect(article).toHaveProperty('title');
+						expect(article).toHaveProperty('article_id');
+						expect(article).toHaveProperty('topic');
+						expect(article).toHaveProperty('created_at');
+						expect(article).toHaveProperty('votes');
+						expect(article).toHaveProperty('comment_count');
+						expect(article.author).toEqual(expect.any(String));
+						expect(article.title).toEqual(expect.any(String));
+						expect(article.article_id).toEqual(expect.any(Number));
+						expect(article.topic).toEqual(expect.any(String));
+						expect(article.created_at).toEqual(expect.any(String));
+						expect(article.votes).toEqual(expect.any(Number));
+						expect(article.comment_count).toEqual(expect.any(Number));
+					});
```

Comment:
better to write a new test for ticket 8 and only test comment_count property

Reply:
This is a new test for the /api/articles end point. It was checking the article appears as it should. I realise that a previous test was checking some of this so have removed all but the comment_count property.